subjects/unzipstring/README.md

```diff
@@ -0,0 +1,59 @@
+## Unzipstring
+
+### Instructions
+
+Write a function called Unzipstring that takes a string that will be a kind of code, and your function will have to decrypt it and return a new string with the output.
```

Comment:
Where are the highlights?
Don't just copy paste my comments because git removes the ` from the words
Missing highlight the func name and lines 12, 13 and 15